### DISCUSSION TITLE
Stop remix `loader` from running after running `action`

### DISCUSSION THREAD

**Discussion by IgnisDa** [remix-run/remix#4921] [Q&A] [answered]
I have a use-case where I am using `fetcher.submit` to submit the form but I do not want the `loader` to run again after successful submission. As far as I can tell, this is not possible yet.

**Comment by machour** [comment, marked as answer]
Take a look at [unstable_shouldReload](https://remix.run/docs/en/v1/route/should-reload)

**Reply by IgnisDa** [reply]
Thanks that worked!
